java/maven/nbproject/project.xml

```diff
@@ -621,6 +621,8 @@
                 <friend>org.netbeans.modules.maven.util</friend>
                 <!-- JShel support -->
                 <friend>org.netbeans.modules.jshell.support</friend>
+                <!-- Micronaut support -->
+                <friend>org.netbeans.modules.micronaut</friend>
```

Comment:
Can this be avoided please? What functionality the micronaut module needs from Maven? Can it be exposed via `java/api.maven/` module instead?

Reply:
Fixed.